### PR TITLE
[Fix-16308] Fix query resources list paging error and docs.

### LINKED ISSUE #16308: [Bug] [Doc] query resources list paging&UDF management not displayed
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

这是我在开发环境 3.2.2-prepared 中遇到的问题，相关服务启动时没有报错，但进入 DS 系统后却不显示 UDF 管理，并报错资源列表查询错误，也无法进行创建文件夹、上传文件等相关操作。
This is the problem I encountered in the development environment 3.2.2-prepared, the relevant services start without any reported error, but after I enter the DS system does not show the UDF management, and reported an error resource list query error, also can not create folders, upload files and other related operations.

### What you expected to happen

我看到配置文件中提到要有\tmp\dolphinscheduler\读写权限，但我第一次完美运行 DS 系统时并没有在意这一点，但后来运行时也没有发现任何错误。
I see that the configuration file says to have \tmp\dolphinscheduler\ read and write permissions, but I didn't care about that the first time I ran the DS system perfectly, but I didn't find any errors on subsequent runs.

### How to reproduce

我是按照官方教程进行开发环境的部署
https://dolphinscheduler.apache.org/zh-cn/docs/3.2.1/contribute/development-environment-setup
I followed the official tutorial to deploy the development environment

### Anything else

However, the first time I Git down DS-3.2.2 and compile and run it there was no such error (no UDF management and folder creation failures), but since I didn't start the alert node successfully the first time, after I re-git it down and compile and run it, I started getting the above problem

### Version

3.2.x

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

close #16308

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/HdfsStorageOperator.java [hunk 1]
```diff
@@ -232,7 +232,11 @@ public List<StorageEntity> listFileStorageEntityRecursively(String resourceAbsol
 
         while (!foldersToFetch.isEmpty()) {
             String absolutePath = foldersToFetch.pollFirst();
-            RemoteIterator<LocatedFileStatus> remoteIterator = fs.listFiles(new Path(absolutePath), true);
+            Path path = new Path(absolutePath);
+            if (!fs.exists(path)) {
+                return result;
```
Comment (ruanwenjun) [inline dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/HdfsStorageOperator.java]: `continue` is better.
```suggestion
                continue;
```

Comment (SbloodyS) [reply dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/HdfsStorageOperator.java]: Fixed.

**Review by Gallardot — APPROVED** [review]
LGTM

**Review by caishunfeng — APPROVED** [review]
+1

**Comment by zhang-dz** [comment]
不显示UDF
创建文件夹失败
为什么我按照你说的去做没有修复问题，我重申一遍我的问题，Windows11启动idea拉取3.2.2版本的源码，通过D:\Ideaproject\dolphinscheduler-dao\src\main\resources\sql\dolphinscheduler_mysql.sql配置dolphinscheduler数据库，修改Api、Master、tool的数据库密码和账号，然后mvn package，然后打开zookeeper启动Api、Master，启动前端npm run dev，访问localhost:5173就出现了上面的问题。
Why didn't I fix the problem by following what you said? I'll reiterate my question, Windows 11 start idea pulls the source code of version 3.2.2, through D:\Ideaproject\dolphinscheduler-dao\src\main\resources\sql\ dolphinscheduler_mysql.sql configure the dolphinscheduler database, change the database password and account of Api, Master, tool, and then mvn package, then open zookeeper to start Api, Master, and start UI npm run dev, access the localhost:5173 and the above problem occurs.

**Comment by zhang-dz** [comment]
其实它创建成功了，我打开D盘发现这个文件是存在的，D:\tmp\dolphinscheduler\default\resources\test7。
It actually created successfully, I opened the D drive and found that the file exists, D:\tmp\dolphinscheduler\default\resources\test7.

**Comment by SbloodyS** [comment]
Currently DS does not support running on Windows. @zhang-dz

**Pull request merged**